src/moin/scripts/maint/modify_item.py

```diff
@@ -33,13 +33,12 @@ def run(self, name, meta_file, data_file, revid):
         item = app.storage.get_item(**fqname.query)
         rev = item[revid]
         meta = json.dumps(dict(rev.meta), sort_keys=True, indent=2, ensure_ascii=False)
-        meta = meta.encode('utf-8')
         meta_lines = meta.split('\n')
         meta_lines = [x.rstrip() for x in meta_lines]
         meta = '\n'.join(meta_lines)
```

Comment:
while you're at it: can you check if lines 36..38 are actually necessary?

almost looks like there was some weirdness in json.dumps() when the code was written (like trailing blanks or wrong line endings?), would be interesting if that is still there in py35+.

Reply:
good catch, deleted the lines